.expeditor/builder_seed.toml

```diff
@@ -2,12 +2,12 @@ format_version = 1
 file_descriptor = "Packages needed to run an instance of Builder"
 
 [[x86_64-linux]]
-channel = "stable"
+channel = "acceptance"
```

Comment:
this should remain `stable` since it is for generating the bootstrap bundle and run after a habitat release wherein we would be promoting to stable.